_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -470,6 +473,10 @@ public function get_all_options() {
 		return rest_ensure_response( $response );
 	}
 
+	public function decode_special_characters( $value ) {
```

Comment:
Even tho it's a useful general-purpose tool, I don't think any outside consumers should use it from this specialised endpoint, so it might make sense to make it `private`, no?

Reply:
👍 - caea42500